src/tests/test_ggplot.py

```diff
@@ -93,6 +93,23 @@ def penguins_no_nulls(ip, penguins_data):
     ).result
 
 
+@pytest.fixture
+def penguins_with_nulls(ip, penguins_data):
+    ip.run_cell(
+        """
+        %sql duckdb://
+        """
+    )
+
+    ip.run_cell(
+        f"""
+%%sql --save with_nulls --no-execute
+SELECT *
+FROM "{penguins_data}"
+    """
+    ).result
```

Comment:
Please add `yield ip` statement at the end

Reply:
Ok will make the change!